opendatahub-io/opendatahub-operator
# Add post check component deployment is ready

## Description

Add post check component deployment is ready
Resolve https://issues.redhat.com/browse/RHOAIENG-349
## How Has This Been Tested?




- Install ODH
- Deploy basic DSC resource
- Check DSC resoruce status

## Merge criteria:
